website/src/segment.css.ts

```diff
@@ -167,3 +167,15 @@ export const infoIcon = style({
 })
 
 export const linkSvg = style({ fill: colors.link })
+
+export const fillerLine = style([
+  {
+    "@media": {
+      [mediaQueries.medium]: {
+        borderBottom: "1px solid black",
+        padding: 0,
+        ...margin(vspace.one, "3rem", vspace.one, 0),
```

Comment:
This will only work on screen sizes larger than mobile. Make sure to try out the mobile version either using the "responsive" feature in the browser dev tools or just shrinking your browser window manually. Then for mobile styles, just add those to the top level. For example:

```ts
style({
  // Mobile and all size styles go here...
  borderBottom: "1px solid black",
  "@media": {
    [mediaQueries.medium]: {
      // Tablet and desktop styles go here...
    }
  }
})
```

Reply:
Also, a small note on styling, we don't want to use a hardcoded color like `black` here. We probably want the line to be the same color as our text, whatever color that may be. So we can use `borderColor: colors.text` instead.